airflow/providers/amazon/aws/operators/ecs.py

```diff
@@ -101,6 +101,10 @@ class ECSOperator(BaseOperator):  # pylint: disable=too-many-instance-attributes
         Only required if you want logs to be shown in the Airflow UI after your job has
         finished.
     :type awslogs_stream_prefix: str
+
+    .. seealso::
```

Comment:
It would be fantastic if the parameter description was at the end of the class description. The link to the guide should be above the description of the parameters. Look how it is done in airflow.providers.google

Reply:
Here is example: https://airflow.readthedocs.io/en/latest/_api/airflow/providers/google/cloud/operators/cloud_storage_transfer_service/index.html#airflow.providers.google.cloud.operators.cloud_storage_transfer_service.CloudDataTransferServiceCreateJobOperator
When this is under the parameters description, it will not fulfill its role, because it will be hardly visible. This will not promote guides.